docker/python3-ubi/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# python3.10.8 at the moment - if updated please update this comment
+ARG BASE_UBI_TAG=8.6
+ARG BASE_UBI_VERSION=ubi8
+
+FROM registry.access.redhat.com/${BASE_UBI_VERSION}:${BASE_UBI_TAG} as build
+
+RUN dnf upgrade -y --nodocs && \
+    dnf install -y --nodocs \
+       bzip2-devel \
+       expat-devel \
+       gcc \
+       libffi-devel \
+       libuuid-devel \
+       make \
+       openssl-devel \
+       sqlite-devel \
+       wget \
+       xz-devel && \
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+
+# install python3
+RUN wget https://www.python.org/ftp/python/3.10.8/Python-3.10.8.tgz && \
+    mkdir -p /usr/local/src/python && \
+    tar -zxf Python-3.10.8.tgz -C /usr/local/src/python --strip-components=1 && \
+    rm -rf Python-3.10.8.tgz && \
+    cd /usr/local/src/python && \
+    ./configure \
+      --enable-loadable-sqlite-extensions \
+      --enable-optimizations \
+      --enable-option-checking=fatal \
+      --with-system-expat \
+      --with-ensurepip && \
+    make && \
+    make altinstall
+
+RUN find /usr/local -depth \
+	\( \
+		\( -type d -a \( -name test -o -name tests -o -name idle_test \) \) \
+		-o \( -type f -a \( -name '*.pyc' -o -name '*.pyo' -o -name '*.a' \) \) \
+	\) -exec rm -rf '{}' + && \
+    echo '/usr/local/lib' > /etc/ld.so.conf && \
+    ldconfig
+
+COPY requirements.txt .
+RUN pip3.10 install -r requirements.txt
+
+FROM registry.access.redhat.com/${BASE_UBI_VERSION}:${BASE_UBI_TAG}
+
+RUN dnf update -y --nodocs && \
+    dnf clean all && \
```

Comment:
Let's also add procps-ng so we have `ps` available for troubleshooting. You can add:
```
dnf install --nodocs -y  procps-ng
```

Reply:
done